Empire/EmpireManager.cpp

```diff
@@ -2,13 +2,25 @@
 
 #include "Empire.h"
 #include "../util/Directories.h"
+#include "../util/GameRules.h"
 #include "../util/Logger.h"
 #include "../util/XMLDoc.h"
 #include "../universe/Enums.h"
 
 #include <boost/filesystem/fstream.hpp>
 
 namespace {
+    void AddRules(GameRules& rules) {
+        // determine if diplomacy allowed
+        rules.Add<std::string>("RULE_DIPLOMACY", "RULE_DIPLOMACY_DESC",
+                               "MULTIPLAYER", "ALLOWED_FOR_ALL", false,
+                               DiscreteValidator<std::string>(std::set<std::string>({
+                                   "RULE_DIPLOMACY_ALLOWED_FOR_ALL",
+                                   "RULE_DIPLOMACY_FORBIDDEN_FOR_ALL"
```

Comment:
Might also make sense to wrap the literals here... not sure what standard practice for this would be as I don't think there are many cases of `DiscreteValidator` with strings...

Reply:
In the `CombatEvents.cpp` nothing is wrapped.

Reply:
Those aren't using `DiscreteValidator` though.

Looking over code a bit, it looks like in (almost?) all cases, calls to `OptionsDB::Add` wrap their string-literal parameters for option name in `UserStringNop`, but in no case I saw does a call to `GameRules::Add` do so. It probably should in all cases, though it doesn't need to be done so for this pull request specifically.